src/main.rs

```diff
@@ -118,10 +118,17 @@ fn run_remove_sequence(args: Args) -> Result<(), Error> {
 	proton_cli::remove_sequence(&admin_key_path, &name)
 }
 
-#[allow(unused_variables)]
 fn run_list_permissions(args: Args) -> Result<(), Error> {
-	let permissions = proton_cli::get_permissions();
-	Ok(println!("{}", permissions.join("\n")))
+	let private_key = args.arg_private_key;
+	if private_key.is_some() {
+		let _ = proton_cli::get_permissions(&private_key.unwrap())
+			.map(|p| println!("{:?}", p));
```

Comment:
It's ugly right now. Thoughts on how you would like to view it? Or should we just return the Vec<Permissions> and let the caller decide how to format it?
`[Permission { which: Administrate, target: None }, Permission { which: EditSeq, target: Some("asdf") }, Permission { which: EditSeqSec, target: Some("glory,1") }]`